cloudify_types/cloudify_types/component/polling.py

```diff
@@ -104,21 +97,27 @@ def redirect_logs(client, execution_id, instance_ctx=None):
                 event.get('message', "")
             )
             message = message.encode('utf-8')
+            level = event.get('level')
 
-            ctx.logger.info(
-                'Message {0} for Event {1} for execution_id {1}'.format(
-                    message, event))
+            # If the event dict had a 'level' key, then the value is
+            # a string. In that case, convert it to uppercase and get
+            # the matching Python logging constant.
+            if isinstance(level, basestring):
```

Comment:
plz avoid basestring checks, and isinstance checks in general. It seems that in this case just a `if level` would work just as well, or perhaps `if level and not isinstance(level, int)` (if it can be int, but the comment says that it'll be a string)

Reply:
Done.